lib/alephant/broker/environment.rb

```diff
@@ -61,7 +61,10 @@ def rack_input
       def parse(json)
         ::JSON.parse(json)
       rescue ::JSON::ParserError => e
-        logger.warn "Broker.environment#data: ParserError"
+        logger.error(
+          method:  'Broker.environment#data',
+          message: 'ParserError'
```

Comment:
Should we log e.message ? We don't seem to use e, even though that may have useful information.